src/crab/splitdbm/graph_ops.hpp

```
@@ -0,0 +1,659 @@
+    std::vector<char> edge_marks;
+    std::vector<VertId> dual_queue;
+    std::vector<int> vert_marks;
+    size_t scratch_sz = 0;
+
+    std::vector<Weight> dists;
+    std::vector<Weight> dists_alt;
+    std::vector<unsigned int> dist_ts;
+    unsigned int ts = 0;
+    unsigned int ts_idx = 0;
+
+    void grow(const size_t sz) {
```

Clang (14.0.6)

[warning] method 'grow' can be made static

(readability-convert-member-functions-to-static)